server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -284,7 +284,7 @@ public void run() {
             if (alog != null) {
               try {
                 alog.close();
-              } catch (Exception e) {
+              } catch (IOException e) {
```

Comment:
This is a change in behavior, but I'm not sure if it's an important one. Avoiding changing the behavior would look like:

```suggestion
              } catch (IOException | RuntimeException e) {
```

Reply:
reverted this change

Reply:
I think it's still helpful to narrow the exception, but it can be done without changing behavior using multi-catch with `| RuntimeException`